### PR TITLE
Statistics from display updater

### PR DESCRIPTION
**Pull request by shanmango**

### CODE REVIEW

**Comment by b1s0ndev** [comment]
Oooo I just realized you made the update for the past hour/day/week, ignore mine on the commit i made

File: js/index.js [hunk 1]
```diff
@@ -150,6 +150,21 @@ function updateArticles(time) {
   }
 }
 
+function updateStatsTimeRangeDisplay(time) {
+  switch(time) {
+    case "hour":
+      $('#stats-time-range').html("Past 60 Minutes");
+      break;
+      case "day":
```
Comment (JimFung) [inline js/index.js]: this is indented one level too deep

File: js/index.js [hunk 2]
```diff
@@ -150,6 +150,21 @@ function updateArticles(time) {
...
+      $('#stats-time-range').html("Past 24 Hours");
+      break;
+      default:
```
Comment (JimFung) [inline js/index.js]: this is also indented 1 level too deep

**Review by JimFung — APPROVED** [review]
LGTM, except for the indents.

**Pull request merged**